source/about/cloud-subscriptions.rst

```diff
@@ -73,6 +64,15 @@ Sales tax and VAT
 
 Mattermost reserves the right to assess applicable taxes as required by local law. Depending on location, you may be charged transaction taxes when purchasing our product. Prices on our website are exclusive of sales tax or VAT.
 
+Failed or late payments
+-----------------------
+
+You'll be notified immediately of failed payments both in-product, with a banner, and via email. The notifications provide directions on how to update payment information. Once payment information is updated, you'll be charged right away for the amount owing from the previous billing period.
+
+If you have not paid or resolved the failed payment after 60 days, your workspace will be downgraded to Mattermost Starter and subject to the data limits of that plan. Your workspace will also be suspended.
+
+Suspended workspaces can be reactivated by contacting Mattermost Customer Support at `https://customers.mattermost.com/cloud/contactus <https://customers.mattermost.com/cloud/contactus>`_. In some cases, suspended workspaces may be deleted. Once a workspace has been deleted, contents can not be reinstated. You'll need to create a new workspace at that time from https://mattermost.com/get-started/.
```

Comment:
In this case, there is no step where a workspace is suspended. The flow is:

Delinquent account -> Auto downgrade to Starter -> If inactive for 60 days, delete workspace.